webapp/packages/plugin-sql-editor/src/SqlEditor/useSqlEditor.ts

```diff
@@ -163,7 +163,9 @@ export function useSqlEditor(state: ISqlEditorTabState): ISQLEditorData {
 
               untracked(() => {
                 this.sqlDialectInfoService.loadSqlDialectInfo(key).then(async dialect => {
-                  await this.updateParserScriptsThrottle();
+                  try {
+                    await this.updateParserScriptsThrottle();
+                  } catch {}
```

Comment:
Do we handle errors inside the parsing function somehow?

Reply:
fixed, now we will log parsing errors from backend within notifications